apiserver/apiserver.go

```diff
@@ -1015,3 +1029,47 @@ func (srv *Server) GetAuditConfig() auditlog.Config {
 	// Delegates to the getter passed in.
 	return srv.getAuditConfig()
 }
+
+func (srv *Server) embeddedCLIHandler(w http.ResponseWriter, req *http.Request) {
+	var err error
+	var result params.StringResults
+	defer func() {
+		if err == nil {
+			err = sendStatusAndJSON(w, http.StatusOK, result)
+		}
+		if err != nil {
```

Comment:
Perhaps had a comment as to why this isn't just an else? It took me a minute to notice, so I think it would help future readers too.